FEATURE_MATRIX.md

```diff
@@ -41,7 +41,7 @@
 | ConversationSummary Skill         | ✅  | ✅     | ❌   |       |
 | FileIO Skill                      | ✅  | ✅     | ❌   |       |
 | Http Skill                        | ✅  | ✅     | ❌   |       |
-| Math Skill                        | ✅  | ✅     | ❌   |       |
```

Comment:
@brunoborges please make this change in a seperate PR against the main branch. We want to the changes to the experimental-java branch to be strictly limited to changes in the java folder so that updating that branch from main won't have merge conflicts.